index.js

```diff
@@ -178,13 +195,15 @@ function compression (options) {
 
       // compression method
       var negotiator = new Negotiator(req)
-      var method = negotiator.encoding(['gzip', 'deflate', 'identity'], ['gzip'])
+      var method = negotiator.encoding(SUPPORTED_ENCODING, PREFERRED_ENCODING)
 
       // if no method is found, use the default encoding
       if (!req.headers['accept-encoding'] && encodingSupported.indexOf(enforceEncoding) !== -1) {
         method = enforceEncoding === '*' ? 'gzip' : enforceEncoding
       }
 
+      console.log(method)
```

Comment:
Looks like a console.log snuck in here. Excited for this feature to land!

Reply:
Good catch, I forgot to remove it while I was debugging here